graylog2-server/src/main/java/org/graylog/events/processor/aggregation/AggregationEventProcessorConfig.java

```diff
@@ -194,37 +203,40 @@ public ValidationResult validate() {
         if (!series().isEmpty() && isConditionsEmpty()) {
             validationResult.addError(FIELD_CONDITIONS, "Aggregation with series must also contain conditions");
         }
+        if (eventLimit() <= 0) {
+            validationResult.addError(FIELD_EVENT_LIMIT, "Event limit must be greater than 0.");
```

Comment:
How does this affect existing event definitions?
Do they need to pick a limit before they can be updated?

Reply:
Yes, that is the current behavior.

Reply:
It would be nice if this wouldn't throw an error. 
We could migrate them to a new upper limit (see below) I suggest `100_000` events. 
We could check with product if that's a good upper boundary

Reply:
changed it